corehq/apps/domain/views/settings.py

```diff
@@ -275,7 +275,7 @@ def logo(request, domain):
     if logo is None:
         raise Http404()
 
-    return HttpResponse(logo[0], content_type=logo[1])
+    return HttpResponse(logo, content_type='image/png')
```

Comment:
Does this still work if the logo is a different type of image, not a png?

Reply:
We always convert and save the logo as a PNG as far as I can tell: https://github.com/dimagi/commcare-hq/blob/b133773ac3ec032a9abe394d88800ede99727aad/corehq/apps/domain/forms.py#L683-L686

Reply:
If there's any concern about that changing in the future and not getting updated in the response object, I could do something closer to the previous code but guess its mimetype from the filename instead of trying to pull from the blob, like 
```python
    def get_custom_logo(self):
        if not self.has_custom_logo:
            return None

        attachment = self.fetch_attachment(LOGO_ATTACHMENT)
        mimetype = mimetypes.guess_type(LOGO_ATTACHMENT)
        return (attachment, mimetype)
```

Reply:
Got it, thank you for explaining. I think this is fine.